internal/elasticsearch/logstash/pipeline.go

```diff
@@ -134,12 +133,7 @@ func ResourceLogstashPipeline() *schema.Resource {
 			Type:        schema.TypeBool,
 			Optional:    true,
 		},
-		"queue_max_bytes_number": {
-			Description: "The total capacity of the queue when persistent queues are enabled.",
-			Type:        schema.TypeInt,
-			Optional:    true,
-		},
-		"queue_max_bytes_units": {
+		"queue_max_bytes": {
 			Description:  "Units for the total capacity of the queue when persistent queues are enabled.",
 			Type:         schema.TypeString,
 			ValidateFunc: validation.StringInSlice([]string{"b", "kb", "mb", "gb", "tb", "pb"}, false),
```

Comment:
```suggestion
			ValidateFunc: validation.Any(
				validation.StringIsEmpty,
				validation.StringMatch(regexp.MustCompile("[0-9]+[bmgtp]?b"), "must be valid size unit"),
			),
```

Might need to add the `regexp` import but I think this should do the right validation.

Reply:
👍 I'll try and take a look at it this week